storybook/src/admin/Tooltip.stories.tsx

```diff
@@ -36,30 +38,116 @@ export const StatusIndicators = {
     },
 };
 
-export const AllVariants = {
+// Makes it easier to see the light variant of the tooltip
+const lightGrayBackgroundDecorator: Decorator = (Story) => (
+    <Box sx={{ backgroundColor: "#f0f0f0", margin: "-40px -30px", padding: "40px 30px" }}>
+        <Story />
+    </Box>
+);
+
+export const StackedTooltipsFromDesign = {
+    decorators: [lightGrayBackgroundDecorator],
     render: () => {
+        const [showTooltips, setShowTooltips] = useState(false);
+
+        useEffect(() => {
+            setTimeout(() => {
+                // Delay showing tooltips to prevent them being rendered in the wrong place due to the underlying element not being rendered properly yet.
+                setShowTooltips(true);
+            }, 1000);
+        }, []);
+
         return (
-            <Stack spacing={6}>
-                <Tooltip title="Dark variant (default)" open placement="right">
-                    <Info />
+            <Stack pb={8} spacing={16} direction="row">
+                <Tooltip
+                    title={
+                        <>
+                            <Typography variant="subtitle2" sx={{ minWidth: 180 }}>
+                                Title
+                            </Typography>
+                            <Typography variant="body2">Notification Text</Typography>
+                        </>
+                    }
+                    variant="light"
+                    placement="bottom-start"
+                    open={showTooltips}
+                >
+                    <Chip label="Light" sx={{ width: 140 }} />
```

Comment:
Do we need to specify the width in a real application? Wouldn't this be dependent on the label's length?